packages.cl

```diff
@@ -107,15 +107,14 @@ v2: version 1.2.58, fix problem introduced in 1.2.56 where the response date
   (require :streamc)
   (require :inflate))
 
-sys::
 (eval-when (compile load eval)
-  (defvar *user-warned-about-deflate* nil)
+  (defvar sys::*user-warned-about-deflate* nil)
   (handler-case (require :deflate)
     (error (c)
-      (if* (null *user-warned-about-deflate*)
+      (if* (null sys::*user-warned-about-deflate*)
 	 then (format t "~&NOTE: ~@<the deflate module could not be loaded, so ~
 server compression is disabled.  AllegroServe is completely functional ~
-without compression.  Original error loading deflate was:~:@>~%~a~%" c)
```

Comment:
According to CLHS:

    If the at-sign modifier is used with ~<...~:>, the entire
    remaining argument list is passed to the directive as its
    argument. All of the remaining arguments are always consumed by
    ~@<...~:>, even if they are not all used by the format string
    nested in the directive.

For Clozure CL, I get this warning:

    ;Compiler warnings :
    ;   In an anonymous lambda form: Too many format arguments in format string:
    ;   "the deflate module could not be loaded, so server compression is disabled.  AllegroServe is completely functional without compression.  Original error loading deflate was:"
    ;                                                                                                                                                                                             ^
    ;   

In SBCL, I get a runtime error:

    error in FORMAT: no more arguments
      ~&NOTE: ~@<the deflate module could not be loaded, so ~
    server compression is disabled.  AllegroServe is completely functional ~
    without compression.  Original error loading deflate was:~:@>~%~a~%

Moving the `~a` into the angle brackets seems to produce the expected result on Allegro CL and works on SBCL and CCL, too.